sdk/storage/storage-file-datalake/package.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name": "@azure/storage-file-datalake",
+  "version": "12.0.0-preview.7",
```

Comment:
Is this by design?

Reply:
Should be preview.6 as the part of track2 preview series.